api4/file.go

```diff
@@ -483,6 +483,10 @@ func getFile(c *Context, w http.ResponseWriter, r *http.Request) {
 		c.Err = err
 		return
 	}
+	if info.DeleteAt != 0 {
+		c.Err = model.NewAppError("getFile", "api.file.get_file.no_file.app_error", nil, "file_id="+info.Id, http.StatusNotFound)
+	}
```

Comment:
Shouldn't there be a `return` here?

Reply:
yes! 🤦 
it still works as the error is carried around, but it's better if we exit early.